common/testdata/romana.sample.yaml

```diff
@@ -14,7 +12,7 @@ services:
     config:
       store: 
         type: sqlite3
-        database: ./ipam.sqlite3
+        database: /tmp/ipam.sqlite3
```

Comment:
Do we really want to put those databases into /tmp, where they couldn't even survive a restart? Might /var/tmp/ be a better place, if we really want it as a temp?